src/binder/assemblybinder.cpp

```diff
@@ -689,11 +689,11 @@ namespace BINDER_SPACE
         sCoreLib = sCoreLibDir;
         sCoreLib.Append(CoreLibName_IL_W);
         BOOL fExplicitBindToNativeImage = (fBindToNativeImage == true)? TRUE:FALSE;
-#if defined(FEATURE_PAL) && !defined(_TARGET_AMD64_) && !defined(_TARGET_ARM64_)
+#if defined(FEATURE_PAL) && !defined(FEATURE_NI_BIND_FALLBACK)
```

Comment:
Semantically, "NI_BIND_FALLBACK" happens when we bind as IL, instead of NI. Thus, the define should look like the following:

`#if defined(FEATURE_PAL) && defined(FEATURE_NI_BIND_FALLBACK)`